src/ripple/protocol/STAmount.h

```diff
@@ -377,15 +377,29 @@ divide (STAmount const& v1, STAmount const& v2, Issue const& issue);
 STAmount
 multiply (STAmount const& v1, STAmount const& v2, Issue const& issue);
 
-// multiply, or divide rounding result in specified direction
+/** Control when bugfixes that require switchover dates are enabled */
+class STAmountCalcSwitchovers
+{
+    bool enableUnderflowFix_ {false};
+  public:
+    explicit
+    STAmountCalcSwitchovers (std::uint32_t parentCloseTime);
+    STAmountCalcSwitchovers () = default;
```

Comment:
I'll remove the default constructor and add a constructor that takes a bool that controls if the switch is enabled or not.